src/components/NoMessageSelected.vue

```diff
@@ -21,22 +21,25 @@
 
 <template>
 	<AppContentDetails>
-		<div id="emptycontent">
-			<IconMail :size="65" />
-			<h2>{{ t('mail', 'No message selected') }}</h2>
```

Comment:
there is some styling for the h2. that is now dead. could you remove it from the bottom of the file?